config/application.rb

```diff
@@ -14,7 +14,7 @@
 module ContentTagger
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.1
+    config.load_defaults 7.0
```

Comment:
Was there an equivalent of the new_framework_defaults file for 7? I have recollections in the past that some (particularly those related to cookies) sometimes need you to run the newer version before changing the defaults.

Be good if this upgrade didn't have as it's a bit of a pain doing the upgrade in two steps.